src/main/java/uk/gov/hmcts/reform/unspec/launchdarkly/OnboardedOrganisation.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.unspec.launchdarkly;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class OnboardedOrganisation {
+
+    @JsonProperty("orgIds")
+    private List<String> orgIds;
```

Comment:
what's the point of this java object don't we just use a list of strings?

Reply:
removed